desktop/src/main/java/bisq/desktop/main/dao/governance/result/VoteResultView.java

```diff
@@ -172,6 +182,57 @@ protected void activate() {
         cyclesTableView.getSelectionModel().selectedItemProperty().addListener(selectedVoteResultListItemListener);
 
         fillCycleList();
+        exportButton.setOnAction(event -> {
```

Comment:
I would prefer to delegate that to a method to keep the activate() method short and easy to understand.
E.g. `exportButton.setOnAction(event -> doExportToJson()`

Reply:
Could you add all fields (if missing, havve not checked all) from the domain objects which are currently not displayed in the UI.
We want to add popups for that anyway and for the json I think its good to have the complete domain data.

The CompensationProposal is missing `requestedBsq` and `bsqAddress`. As said the UI does not show all that yet but would be good to have that in the json.

Reply:
For Proposal there are a few subclasses with diff. fields...